modules/ROOT/pages/create-instance-task-mule.adoc

```diff
@@ -6,18 +6,14 @@ endif::[]
 
 include::partial$task-add-api-instance.adoc[tags=intro1;mule-intro;intro2]
 
+[[add-api]]
```

Comment:
What does this mean?

Reply:
This is an anchor, so it allows the user when they click on the page to go down the specific section on adding an API.